building/windows.md

```diff
@@ -0,0 +1,231 @@
+for your system.
+
+![OS bit selection](_images/qemu-os-select.png)
+
+Next up we need to download `Cygwin` from the [official website](https://cygwin.com/install.html). It is a collection of
+Open Source tools that provide
+
+<details>
+<summary>`Cygwin` installation/update process (click to expand)</summary>
+&nbsp;
+
+* The installation process also serves as a package manager. `Cygwin` will ask about a type of installation in our case
+it will be from the internet.
+
+![cygwin-download-source](_images/cygwin-download-source.png)
+
+* After that, the installation process will ask about the placement of a root and local package directory
+
+![cygwin-root-dir](_images/cygwin-root-dir.png)
+![cygwin-local-directory](_images/cygwin-local-directory.png)
+
+* `Cygwin` has the capability of using host's internet connection, but it needs to be configured with our preferences
+(In most scenario it will be System Proxy Settings)
+
+![cygwin-internet-connection](_images/cygwin-internet-connection.png)
+
+* Selection of a mirror for package downloads can be tricky, sometimes it needs to be changed for better results because
+not all mirror have all available packets.
+
+![cygwin-mirror-selection](_images/cygwin-mirror-selection.png)
+
+* After selecting the mirror we can select the packages we want. But in larger groups, this can be annoying. That's why
+we only select two elements (They will be installed with dependencies) which are mandatory to use package manager
+`apt-cyg` in `Cygwin` terminal.
+
+![cygwin-selecting-packages](_images/cygwin-selecting-packages.png)
+
+* Once these are chosen, proceed to run the installation process.
+
+</details>
+  &nbsp;
+
+If installation process was performed, next step is to run `Cygwin` terminal.
+Always run it as administrator, the best way to do that is simply setup this as default action:
+
+![cygwin-admin-rights](_images/cygwin-admin-rights.png)
+
+  <details>
+<summary>Installing apt-cyg and mandatory packages (click to expand)</summary>
+
+  &nbsp;
+<details>
+<summary>Where place listed commands?</summary>
+
+  After lunch `Cygwin` terminal just perform command:
+  
+  ```bash
+  cd
+  ```
+
+  This will move you to default directory placed in `../cygwin64/home/admin/` on Windows.
+
+  In `cygwin` terminal is pointed as `~` and it determinate you are in main directory. This will look like:
+
+  ![cygwin-cmd-main-dir](_images/cygwin-cmd-main-dir.png)
+
+</details>
+  &nbsp;
+
+``` bash
+lynx -source rawgit.com/transcode-open/apt-cyg/master/apt-cyg > apt-cyg && \
+install apt-cyg /bin && \
+apt-cyg install libhidapi-devel libgccpp1 libtool \
+libusb-devel libvirt-client libvirt-devel \
+mingw64-i686-gcc-core mingw64-i686-gcc-g++ \
+mingw64-x86_64-gcc-core mingw64-x86_64-gcc-g++ \
+gcc-core gcc-g++ libclang-devel nano vim \
+patch picocom python3 python3-pip python3-sphinx \
+python3-devel qemu-integration symlinks \
+tty-clock usbutils virt-viewer make autoconf \
+automake cmake libc-client-devel git

```

Comment:
Perhaps it would be worth adding a line like 'Run this in the terminal' before the commands

Reply:
```
<summary>Where place listed commands?</summary>
```
It is mentioned